recipes/mdnsd/mdns_544.oe

```diff
@@ -76,16 +82,9 @@ pkg_prerm_${PN} () {
         -i $D/etc/nsswitch.conf
 }
 
-FILES_${PN} += "${libdir}/libdns_sd.so.1 \
-                ${bindir}/dns-sd \
-                ${libdir}/libnss_mdns-0.2.so \
-                ${sysconfdir}/nss_mdns.conf"
-
-FILES_${PN}-dev += "${libdir}/libdns_sd.so \
-                    ${includedir}/dns_sd.h "
-
-FILES_${PN}-man += "${mandir}/man8/mdnsd.8 \
-                    ${mandir}/man5/nss_mdns.conf.5 \
-                    ${mandir}/man8/libnss_mdns.8"
-
-PACKAGES = "${PN} ${PN}-dev ${PN}-man ${PN}-dbg"
+FILES_${PN}-libnss-mdns-dev = ""
+FILES_${PN}-dev = ""
+FILES_${PN}-libnss-mdns += "${libdir}/libnss_mdns-0.2.so ${sysconfdir}/nss_mdns.conf"
+FILES_${PN}-libdns-sd-dev = "${includedir}/dns_sd.h"
+FILES_${PN}-libdns-sd += "${libdir}/libdns_sd.so"
```

Comment:
yes, the .so and .so.1 files do not contain version information as we normaly see with libraries made with e.g. libtool. Linking with -l results in a dependency on the .so file and not the .so.1. Therefore the .so is included in the ${PN}-libdns-sd package. It might be solved later with some manual libtool install(?) on the library or similar.